site/description.html

```diff
@@ -21,4 +21,13 @@
 activity books, exercise banks, research findings, and more, made available
 to you as free and open-source resources for your classroom.
 </p>
-<a class="button" href="/pages/about.html">Learn more!</a>
+<p style="text-align:center">
+    <a class="button" href="./about/">Learn more!</a>
+</p>
+<p>
+We gratefully acknowledge support provided by the National Science Foundation,
+<a href="https://nsf.gov/awardsearch/showAward?AWD_ID=2011807">Award #2011807</a>.

```

Comment:
```suggestion
This material is based upon work supported by the National Science Foundation under
<a href="https://nsf.gov/awardsearch/showAward?AWD_ID=2011807">Award #2011807</a>.
Any opinions, findings and conclusions or recommendations expressed in this material are those 
of the author(s) and do not necessarily reflect the views of the National Science Foundation.
```

There is [specific language](https://new.nsf.gov/policies/pappg/24-1/ch-11-other-post-award-requirements#ch11E4) in the PAPPG we are supposed to use